test/integration/admin_login_test.rb

```diff
@@ -192,7 +192,7 @@ def setup
     @item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200, retired: false)
 
     visit admin_items_path
-    select 'true', from: 'retire[status]'
+    select 'true', from: "retire[status_#{@item.id}]"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.